testsuites/CBLTester/CBL_Functional_tests/conftest.py

```diff
@@ -427,7 +429,8 @@ def params_from_base_suite_setup(request):
         "device_enabled": device_enabled,
         "flush_memory_per_test": flush_memory_per_test,
         "delta_sync_enabled": delta_sync_enabled,
-        "enable_file_logging": enable_file_logging
+        "enable_file_logging": enable_file_logging,
+        "suite_db_log_files": suite_db_log_files
```

Comment:
suite_db_log_files is introduced here but not used anywhere?

Reply:
Yes, I added for future reference. I can remove it if not needed.